bin/apiml_cm.sh

```diff
@@ -319,34 +372,62 @@ function jwt_key_gen_and_export {
     -file ${SERVICE_KEYSTORE}.${JWT_ALIAS}.pem
 }
 
+#function trust_zosmf {
```

Comment:
> How can be the user sure or validate that it is really z/OSMF certificate?

Fingerprint and subject of the pulled certificates will be printed to console and user will have to check if they are correct. This will be mentioned in doc